src/android/com/adobe/phonegap/push/GCMIntentService.java

```diff
@@ -229,6 +229,21 @@ private Bundle normalizeExtras(Context context, Bundle extras) {
                         Log.e(LOG_TAG, "normalizeExtras: JSON exception");
                     }
                 }
+            } else if (key.equals(APPCELERATOR_PAYLOAD)) {
+                try {
+                    String json = extras.getString(key);
```

Comment:
So AppCelerator sends it's payload as a double encoded JSON string? That's weird but Amazon SNS does something similar.